cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/common/BatchFileFilter.java

```diff
@@ -80,7 +80,8 @@ public boolean accept(Path path) {
 
     //filter by file name using regex from configuration
     if (!useTimeFilter) {
-      Matcher matcher = regex.matcher(filePathName);
```

Comment:
The regex is meant only to match against the last component of the file Path, or the entire file path?